### PR TITLE
integrations for mongoose use

### PR DESCRIPTION
**Pull request by benrkaplan**

### CODE REVIEW

File: back-end/app.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 const express = require('express')
+const mongoose = require('mongoose')
```
Comment (sam-eng) [inline back-end/app.js:2]: Could you make sure to include mongoose in the dependencies in package.json?

**Review by sam-eng — CHANGES_REQUESTED** [review]

File: back-end/app.js (hunk 1 above)
Comment (sam-eng) [reply back-end/app.js:2]: Other than this, it looks good!

Comment (benrkaplan) [reply back-end/app.js:2]: done

**Review by LevBernstein — APPROVED** [review]
Looks good, thanks for making those changes

**Pull request merged**